source/includes/gcp/_health_checks.md

```diff
@@ -1,5 +1,92 @@
 ### Health checks
 
+<!-------------------- List HEALTH CHECK -------------------->
+#### List health Checks
+
+```shell
+curl --request GET \
+  --url http://cloudmc_endpoint/api/v2/services/your_service/your_environment/healthchecks \
+  --header 'mc-api-key: your_api_keys'
+  # The above command returns JSON structured like this:
+```
+
+```json
+{
+  "data": [
+    {
+      "checkIntervalSec": 10,
+      "creationTimestamp": "2019-09-18T12:39:48.982-07:00",
+      "description": "",
+      "healthyThreshold": 2,
+      "id": "5930212998788364011",
+      "kind": "compute#healthCheck",
+      "name": "firsthealthcheck",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/cmc-gcp-env-jamie-hvh/global/healthChecks/firsthealthcheck",
+      "tcpHealthCheck": {
```

Comment:
Please don't document a tcpHealthCheck, we only support HTTP and HTTPS for now.

Reply:
@JamieWulala you still haven't fix this. Please provide an example which has a `httpHealthCheck` or `httpSHealthCheck`